cristin-intermediate-storage/src/test/resources/cristinUnits/units7.json

```diff
@@ -0,0 +1,5320 @@
+[ {
```

Comment:
where does this data come from and how do we maintain it?

Maybe add a README or something?

Reply:
see comment over